modules/gdscript/tests/scripts/analyzer/warnings/get_node_without_onready.gd

```diff
@@ -1,18 +1,30 @@
 extends Node
 
-var add_node = do_add_node() # Hack to have one node on init and not fail at runtime.
+var add_node = do_add_nodes() # Hack to have nodes on init and not fail at runtime.
 
 var shorthand = $Node
 var with_self = self.get_node(^"Node")
 var without_self = get_node(^"Node")
 var with_cast = get_node(^"Node") as Node
 var shorthand_with_cast = $Node as Node
 
+var shorthand_unique = %UniqueNode
+var shorthand_in_dollar_unique = $"%UniqueNode"
+var without_self_unique = get_node(^"%UniqueNode")
+var shorthand_with_cast_unique = %UniqueNode as Node
```

Comment:
```suggestion
```

Reply:
In my opinion these cases are redundant, we use the same logic for regular and unique node shorthands.

Reply:
IMO it can still make sense, as currently we use the same logic but if this changed in the future, the unit test would help ensure the behavior stays consistent. But I don't feel strongly about it, removing it is also fine.

Reply:
I'd say it doesn't harm either way so let's not block this too long, and get it tested in the next beta.